### PR TITLE
Music schema

### PR DESCRIPTION
**Pull request by chimwemwe007**
### The following were completed in this pull request.

- Create schema for music album.
- Create schema for genres.

### CODE REVIEW

File: schema/schema.sql
Comment (Ney2) [inline schema/schema.sql:4]: The foreign key is the one that relates to tables so instead of using 'references item(id)' it should be 'references geners(id)'

File: schema/schema.sql
Comment (Ney2) [inline schema/schema.sql:1]: Music album class has inputs like publish date and archived so kindly add these attributes in the schema

**Review by Ney2 — CHANGES_REQUESTED** [review]
## Status  Required Change ♻️ 
Good job so far. You are almost there to go to your next project.
Kindly see the comment section below

Happy coding

**Comment by bengigo** [comment]
I have started the second review

**Review by bengigo — APPROVED** [review]
# Status Approved ✔️✔️✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: schema/schema.sql
Comment (chimwemwe007) [reply schema/schema.sql]: hello, this has been solved.

**Comment by chimwemwe007** [comment]
Thank you.

**Pull request merged**
